src/app/pages/data-protection/replication/replication-wizard/replication-wizard.component.ts

```diff
@@ -570,7 +570,7 @@ export class ReplicationWizardComponent implements WizardConfiguration {
           width: '50%',
         },
         {
-          placeholder: helptext.lifetime_value_placeholder,
+          placeholder: '',
```

Comment:
We're setting placeholder to empty string? Previously it was `'Snapshot Lifetime'`